meteor/client/mozdef.js

```diff
@@ -5,19 +5,13 @@ file, You can obtain one at http://mozilla.org/MPL/2.0/.
 Copyright (c) 2014 Mozilla Corporation
 */
 import { Meteor } from 'meteor/meteor';
-import { Template } from 'meteor/templating';
 import validator from 'validator';
-import '/imports/collections.js';
-import '/imports/settings.js';
-import '/imports/helpers.js';
-import '/client/js/jquery.highlight.js';
-import PNotify from 'pnotify';
 import 'pnotify/dist/pnotify.css';
-import './mozdef.html';
-import './menu.html';
-import './menu.js';
-import '/client/layout.js';
-import '/public/css/dropdowns.css';
+import PNotify from 'pnotify';
+// import './mozdef.html';
```

Comment:
Should we delete these lines?

Reply:
yes, jeff commented these out, was probably due for removal if this PR got approved.

Reply:
removed in last commit.